components/Styling/StyledForm.js

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components";
+
+export const FormContainer = styled.form`
+  width: 95%;
+  margin: 20px;
+  padding: 20px 0px;
+  background-color: ${(props) => `var(--${props.backgroundcolor})`};
+  // background-color: var(--globalPlaceBackgroundColor);
+  border-radius: var(--globalborderradius);
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
```

Comment:
I think `flex-start` is the default value of justify-content. This CSS declaration can be removed I think.